integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -214,6 +219,9 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
               e -> {
                 throw new WebClientResponseWithMessageException(e);
               })
+          .onErrorResume(
+              WebClientRequestException.class,
+              e -> Mono.error(new WebClientRequestWithMessageException(e)))
```

Comment:
Why do we use `throw new ...` on L220 but `Mono.error()` on L224? I mostly see the `Mono.error()` approach, is L220 wrong?

Reply:
it should be `Mono.error()`, can update this in another PR as well